tests/snappi_tests/intf_utils/intf_accept_metrics.py

```diff
@@ -0,0 +1,147 @@
+# This file defines the interfaces that snappi tests accept external metrics.
```

Comment:
All common label names are missing too, e.g.: PortId, QueueId, PSUId....

otherwise it will be very hard to create unified dashboard, because each tests could use its own names, and causing problems in filters.